commons-vfs2/src/main/java/org/apache/commons/vfs2/FileContent.java

```diff
@@ -147,6 +147,22 @@
      */
     InputStream getInputStream() throws FileSystemException;
 
+    /**
+     * Returns an input stream for reading the file's content.
+     * <p>
+     * There may only be a single input or output stream open for the file at any time.
+     *
```

Comment:
In Java 8 you must close tags, so a paragraph lives in a **`<p></p>`** block. IIRC, the Javadoc treats the first sentence of a Javadoc differently and it does not need to live in such a block.

Reply:
Done.